src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -118,8 +133,8 @@ public void visit(NodeTraversal t, Node n, Node parent) {
                 new Node(Token.EXPORT_SPEC, Node.newString(Token.NAME, localName))));
             parent.addChildAfter(export, n);
             // Registers symbol for rewriting local uses.
-            registerLocalSymbol(child.getSourceFileName(), exportedNamespace, exportedNamespace,
-                localName);
+            registerLocalSymbol(
```

Comment:
These should only be lines that I touched, rather than the whole file.  I probably made a change then reverted it, but the changes were likely in different git sub-commits so it didn't quite trigger right.

That being said, I can send a "reformat cl," if you'd like the code base as a whole to match google-java-format.